README.md

```diff
@@ -1,8 +1,142 @@
 # trading_calendars
 
 ![CI](https://github.com/quantopian/trading_calendars/workflows/CI/badge.svg)
+[![PyPI version](https://img.shields.io/pypi/v/trading-calendars.svg)](https://pypi.org/project/trading-calendars/)
+[![Conda version](https://img.shields.io/conda/vn/conda-forge/trading-calendars.svg)](https://anaconda.org/conda-forge/trading-calendars)
 
-A Python library of exchange calendars meant to be used with [Zipline](https://github.com/quantopian/zipline).
+A Python library of exchange calendars, frequently used with [Zipline](https://github.com/quantopian/zipline).
+
+
+## Installation
+
+```bash
+$ pip install trading-calendars
+```
+
+## Quick Start
+
+```python
+import trading_calendars as tc
+import pandas as pd
+import pytz
+```
+
+Get all registered calendars with `get_calendar_names`:
+
+```python
+>>> tc.get_calendar_names()[:5]
+['XPHS', 'FWB', 'CFE', 'CMES', 'XSGO']
+```
+
+Get a calendar with `get_calendar`:
+
+```python
+>>> xnys = tc.get_calendar("XNYS")
+```
+
+Working with sessions:
+
+```python
+>>> xnys.is_session(pd.Timestamp("2020-01-01"))
+False
+>>> xnys.next_open(pd.Timestamp("2020-01-01"))
```

Comment:
How about an empty line between each pair of input/output?

Certainly subjective, but denser than my liking currently:
<img width="470" alt="image" src="https://user-images.githubusercontent.com/2123660/95791037-84fdcb00-0cae-11eb-9ea2-bae41de7407f.png">